source/whats-new.txt

```diff
@@ -82,7 +82,12 @@ To learn more, see the `v4.12.0 Release Highlights <https://github.com/mongodb/n
 What's New in 4.11
 ------------------
 
-The 4.11 {+driver-short+} release includes added support for **mutually
+When using the 4.11 {+driver-short+} with MongoDB Server version 6.0 or
```

Comment:
Suggestion:
- Since this is in the 4.11 header, I think it is redundant to mention the driver version (in this case). It's also redundant in the other versions as well, but that's outside the scope of this PR.
- The phrase "note that" doesn't seem necessary when introducing information.
- I try to avoid the "with" preposition whenever possible since you can usually interpret it multiple ways and can therefore be ambiguous. Instead, this could describe the action -- "When connecting to MongoDB Server v6.0 or later, the driver"